### DISCUSSION TITLE
Delete the app from Github

### DISCUSSION THREAD

**Discussion by clottedcode** [Digitaler-Impfnachweis/covpass-android#233] [Ideas]
The app reminds me of Nazi Germany during World War II, where travel documents were common practice. The idea itself is awful and facilitates discrimination on a national level.

**Comment by Ein-Tim** [comment]
@wkornewald @pucherd @alexcimander

Please close & lock this thread. This is IMO a violation of the Code of Conduct.

See also how the CWA-Team reacted to something like this: https://github.com/corona-warn-app/cwa-wishlist/issues/651

**Comment by carlolf** [comment]
It is not wise to remind to Nazi Germany - you will soon be regarded as idiot. Better talk of China's apps controlling access to supermarkets, dance floors, trains and airplanes. The big brothers watching and controlling you. This app here and similiars in western world obviously goes into this direction. Covid-19 (with all it's false narratives) is just a pretext.

**Reply by clottedcode** [reply]
It's scary to witness you facilitating discrimination in Europe and even talking it good (because China does it too). China has no morals and ethics whatsoever. And you think the EU has to go in that direction. Why? I want the app to be removed from Github because it is unethical, simple as that. The EU constitution disallows discrimination, otherwise the EU itself is corrupt. I will not stop until the evil app is gone.

**Reply by Ein-Tim** [reply]
@clottedcode 

Maybe you have valid arguments, maybe not. However, the correct way to express your opinion and make use of your freedom of speech right is not this development platform in my opinion.

Also, I have to add, it is unacceptable to compare this app to the nazi regime. This is just unacceptable & disrespectful, to the victims & the relatives of the victims of the Holocaust and other crimes committed during this time.

I can only urge the CovPass-Team, again, to lock this thread.

@wkornewald @G00fY2 @pucherd @alexcimander
